### ISSUE TITLE
`inputs` context not supported for `workflow_dispatch`

### ISSUE THREAD

**Issue by joshbolduc** [rhysd/actionlint#152]
Thanks for providing this helpful tool!

GitHub [recently added support for referencing the `inputs` context with `workflow_dispatch`](https://github.blog/changelog/2022-06-10-github-actions-inputs-unified-across-manual-and-reusable-workflows/), in addition to `github.event.inputs`. The [docs now present `inputs` as the primary way to retrieve input passed to `workflow_dispatch`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs) with this note:

> **Note:** The workflow will also receive the inputs in the `github.event.inputs` context. The information in the `inputs` context and `github.event.inputs` context is identical except that the `inputs` context preserves Boolean values as Booleans instead of converting them to strings.

So a workflow file like the following:
```yml
name: Example

on:
  workflow_dispatch:
    inputs:
      my_input:
        type: string
        required: true

jobs:
  my_job:
    runs-on: ubuntu-latest
    steps:
      - if: ${{ github.event.inputs.my_input }}
        run: echo test
      - if: ${{ inputs.my_input }}
        run: echo test
```

should work, but actionlint reports this error:
```
example.yml:16:17: property "my_input" is not defined in object type {} [expression]
   |
16 |       - if: ${{ inputs.my_input }}
   |    
```

**Comment by Rosswell** [comment]
just as another data point, I am running into the same issue when defining env vars as inputs

**Comment by andrewthetechie** [comment]
Same issue here as well

Workflow: https://github.com/andrewthetechie/gha-cookiecutter/blob/main/.github/workflows/release.yml

Output: 
```.github/workflows/release.yml:39:36: property "version" is not defined in object type {} [expression]
.github/workflows/release.yml:66:40: property "version" is not defined in object type {} [expression]
.github/workflows/release.yml:73:32: property "version" is not defined in object type {} [expression]
   |
73 |           tagging_message: ${{ inputs.version }}
   |                                ^~~~~~~~~~~~~~
.github/workflows/release.yml:84:26: property "version" is not defined in object type {} [expression]
   |
84 |           tag_name:  ${{ inputs.version }}
   |                          ^~~~~~~~~~~~~~
.github/workflows/release.yml:85:30: property "version" is not defined in object type {} [expression]
   |
85 |           release_name:  ${{ inputs.version }}
   |                              ^~~~~~~~~~~~~~
.github/workflows/release.yml:86:22: property "releasenotes" is not defined in object type {} [expression]
   |
86 |           body:  ${{ inputs.releaseNotes }}
   |                      ^~~~~~~~~~~~~~~~~~~
.github/workflows/release.yml:88:27: property "prerelease" is not defined in object type {} [expression]
   |
88 |           prerelease: ${{ inputs.prerelease }}
   |                           ^~~~~~~~~~~~~~~~~
(gha-repo-manager) ➜  gha-repo-manager git:(main) ✗ actionlint --version
1.6.13
installed by building from source
built with go1.18.2 compiler for darwin/arm64```

**Comment by rhysd** [comment]
Thanks, this should be fixed.

When `workflow_dispatch` event was implemented in GitHub Actions, only `github.event.inputs` value was set and `inputs` context was not set. Now `inputs` context is also set.

**Comment by rhysd** [comment]
This fix was included in the latest release [v1.6.14](https://github.com/rhysd/actionlint/releases/tag/v1.6.14).

**Comment by andrewthetechie** [comment]
Confirmed this fixed my issue. Thanks for the quick fix!
